features/step_definitions/data_set_steps.rb

```diff
@@ -115,8 +137,16 @@
   save_and_open_page
 end
 
-Then /^I fill in the form to create the "(.*?)" service$/ do |name|
-  fill_in_form_with(name, csv_path_for_data(name))
+When /^I fill out the form with the following attributes to create a service:$/ do |table|
+  params = table.rows_hash.symbolize_keys!
+  assert params[:name]
```

Comment:
Is this a legitimate thing to do? I want to ensure that params[:name] has been set.